src/EnrollChoicesController.js

```diff
@@ -47,13 +47,26 @@ function (Okta, FormController, Enums, RouterUtil, FactorList,
         case Enums.ALL_OPTIONAL_SOME_ENROLLED:
         case Enums.HAS_REQUIRED_ALL_REQUIRED_ENROLLED:
           return Okta.loc('enroll.choices.optional', 'login');
+        case Enums.HAS_REQUIRED_SOME_REQUIRED_ENROLLED:
+          const remainingDays = this.options.appState.get('gracePeriodRemainingDays');
+          return (remainingDays >= 0) ? this.getGracePeriodSubtitle(remainingDays) : this.getDefaultSubtitle();
```

Comment:
if gracePeriodRemainingDays is set to null when it is not defined. 

null >= 0   gives => true 

We need to update the logic here.

Reply:
Yup, changed this. Will push soon